### ISSUE TITLE
Permanently broken/removed links in doc

### ISSUE THREAD

**Issue by johanandren** [akka/akka#31089]
Figure out what to do, then replace/remove them.

**Comment by muskan3006** [comment]
Hi I checked the conf file, we have to fix no-https whitelist?
@johanandren

**Comment by johanandren** [comment]
Yes, a few, but not all in the `non-https-whitelist` are sites/research papers that are simply not there anymore, for each of those we need to figure out if we can find the paper elsewhere or remove/change that page in the docs to not have a link anymore.
